treeherder/config/settings.py

```diff
@@ -159,6 +159,9 @@
         # truncated upon insertion, so Django highly recommends activating a strict mode for MySQL to
         # prevent data loss (either STRICT_TRANS_TABLES or STRICT_ALL_TABLES).
         'init_command': "SET sql_mode='STRICT_TRANS_TABLES'",
+        # Ensure we read the value inside the transaction, not some other volatile value
+        # https://docs.djangoproject.com/en/2.1/ref/databases/#mysql-isolation-level
+        'isolation_level': 'serializable',
```

Comment:
IMPORTANT: This will ensure the data read from the database will not be changed by another transaction.  It may slowdown all transactions

Reply:
I would wait for [bug 1603249](https://bugzilla.mozilla.org/show_bug.cgi?id=1603249) to land first.

How did the run in prototype go?
You can use New Relic to compare the data ingestion between that app and stage (those two apps have the same EC2 instance size and similar IOPS). You can put New Relic side by side for both apps and look at transactions by the slowest ones.

Reply:
> I would wait for bug 1603249 to land first.

I am in no rush to land this

> How did the run in prototype go?

@armenzg  I did not know that prototype and stage could be compared.  I will deploy to prototype later this week and report my findings back here.  Thank you!